gradle/wrapper/gradle-wrapper.properties

```diff
@@ -2,4 +2,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.6-bin.zip
```

Comment:
I was forced to increase Gradle because of

> FAILURE: Build failed with an exception.
> 
> * Where:
> Build file '/Users/hannes/git/AntennaPod/core/build.gradle' line: 1
> 
> * What went wrong:
> A problem occurred evaluating project ':core'.
> > Failed to apply plugin [id 'com.android.library']
>    Configuration on demand is not supported by the current version of the Android Gradle plugin since you are using Gradle version 4.6 or above. Suggestion: disable configuration on demand by setting org.gradle.configureondemand=false in your gradle.properties file or use a Gradle version less than 4.6.